snap/Dockerfile.aarch64

```diff
@@ -0,0 +1,15 @@
+FROM arm64v8/ubuntu:xenial
```

Comment:
Should this file be named Dockerfile.arm64?

Reply:
oh no, does `uname -m` return aarch64 for arm64?

Reply:
yea :\